invenio_app_ils/ext.py

```diff
@@ -186,6 +204,24 @@ def init_app(self, app):
         # disable warnings being logged to Sentry
         logging.getLogger("py.warnings").propagate = False
 
+    def init_metadata_extensions(self, app):
+        """Metadata extensions initialization."""
+        allowed_types = ["document", "series"]
```

Comment:
shouldn't this be in the config? ping, @ntarocco

Reply:
I think this should be completely removed if favor of below?